### DISCUSSION TITLE
Migrate from LAUNCHXL- CC26x2R1 from Pi to LAUNCHXL- CC1352P on NUC

### DISCUSSION THREAD

**Discussion by salopette** [Koenkk/zigbee2mqtt#8477] [General]
Hello, is a change possible without re-pairing devices?
Is it enough to copy the complete folder (data), or does the Pan-ID still have to be changed?

thanks

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/information/FAQ.html#what-does-and-does-not-require-repairing-of-all-devices

**Reply by salopette** [reply]
> See https://www.zigbee2mqtt.io/information/FAQ.html#what-does-and-does-not-require-repairing-of-all-devices

Hello, thanks, I've read that, but I don't understand it correctly, maybe Google translates it strangely.
That's why I wanted the 100% answer here.

Thanks again!

**Reply by Koenkk** [reply]
no repairing is needed, just plug in the new coordinator and update the `port` (location)

**Comment by salopette** [comment]
Cool! Do I have to copy the entire data folder? Or just the configurations.yaml?

**Reply by Koenkk** [reply]
entire data folder
